@xen-orchestra/web/src/components/backups/jobs/panel/cards/BackupJobInfosCard.vue

```diff
@@ -18,17 +18,15 @@
           <VtsCopyButton :value="backupJob.id" />
         </template>
       </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('mode') }}</template>
+      <VtsCardRowKeyValue v-for="(mode, index) in backupJobModes" :key="mode">
+        <template #key>
+          <template v-if="index === 0">{{ t('mode', backupJobModes.length) }}</template>
+        </template>
         <template #value>
-          <UiTagsList>
-            <UiTag v-for="(mode, index) in backupJobModes" :key="index" accent="info" variant="secondary">
-              {{ mode }}
-            </UiTag>
-          </UiTagsList>
+          <span class="text-ellipsis">{{ mode }}</span>
         </template>
-        <template v-if="backupJobModes !== undefined" #addons>
-          <VtsCopyButton :value="backupJobModes.join(', ')" />
+        <template #addons>
+          <VtsCopyButton :value="mode" />
         </template>
```

Comment:
```suggestion
        <template #addons>
          <VtsCopyButton :value="mode" />
                    <UiButtonIcon
            v-if="index === 0 && backupJobModes.length > 1"
            v-tooltip="t('coming-soon')"
            disabled
            icon="fa:ellipsis"
            size="medium"
            accent="brand"
          />
</template>
```

For corresponding with Figma and other components (like ip on `hostPifSidePanel`), maybe add an ellipsis icon on the first value.